src/solutions/ode_solutions.jl

```diff
@@ -604,3 +604,17 @@ function sensitivity_solution(sol::ODESolution, u, t)
     @reset sol.t = t isa Vector ? t : collect(t)
     return @set sol.interp = interp
 end
+
+function strip_solution(sol::ODESolution)
+    if has_lazy_interpolation(sol.alg)
+        error("The algorithm $(sol.alg) uses lazy interpolation, which is incompatible with
+        solution stripping.")
+    end
+
+    interp = strip_interpolation(sol.interp)
```

Comment:
@jClugstor : Just randomly reviewing this. It seems strange to me that the `interp` variable isn't used afterwards? Should it go into line 618?

Reply:
Yes that looks like a typo.

Reply:
Yeah, it's a typo, thanks for pointing it out .